### DISCUSSION TITLE
webdav增加“名称”

### DISCUSSION THREAD

**Discussion by nxjonson** [maotoumao/MusicFree#366] [Q&A]
alist挂载网盘支持webdav，网盘内存了大量的音乐，目前插件webdav只能设定一个指向，建议webdav增加名称，可以设定多个指向。

**Comment by nxjonson** [comment]
仔细阅读了webdav使用方法，解决了：存放路径用英文的逗号分开就可以。

**Reply by oldsento** [reply]
学到了，具体原来是这样，比如/baiduyunan/俺音乐/预/,/baiduyunan/俺音乐/伴奏(1),/baiduyunan/俺音乐/纯音乐,/baiduyunan/俺音乐/
